Formula/buku.rb

```diff
@@ -23,6 +23,6 @@ def install
   test do
     ENV["XDG_DATA_HOME"] = "#{testpath}/.local/share"
     system "#{bin}/buku", "-a", "https://github.com/Homebrew/homebrew"
-    assert_match %r{https://github.com/Homebrew/homebrew}, shell_output("#{bin}/buku -s github </dev/null")
+    assert_match %r{https://github.com/Homebrew/homebrew}, shell_output("#{bin}/buku --noprompt -s github")
```

Comment:
Can you also update this to use a string instead of a regular expression? I don't think we need to interpret the `.` between `github` and `com` to mean “any character”. Thanks!